webapp/graphite/util.py

```diff
@@ -200,6 +200,9 @@ class SafeUnpickler(pickle.Unpickler):
       'graphite.intervals': set(['Interval', 'IntervalSet']),
     }
 
+    def __init__(self, file):
+        super().__init__(file, encoding='bytes')
```

Comment:
```suggestion
        super().__init__(file, encoding='utf8')
```

Setting this to `utf8` instead fixes the issue. When set to `bytes` no results are displayed, even ascii results.